core/translations/i18n/en.json

```diff
@@ -49,82 +49,7 @@
   "Unloading module \"${0}\".": "Unloading module \"${0}\".",
   "What kind of shit platform is this that doesn't even support sending messages?": "What kind of shit platform is this that doesn't even support sending messages?",
   "Working on it...": "Working on it...",
-
```

Comment:
tbh I don't think I have the willpower to go through 40 repositories just to rename a file....